client/sites/staging-site/components/staging-site-sync-modal/index.tsx

```diff
@@ -399,112 +399,117 @@ export default function SyncModal( {
 					<Tooltip
 						text={
 							shouldDisableGranularSync
-								? __( 'Selective Sync will be enabled automatically once your backup is complete.' )
+								? __(
+										'Selecting individual items to sync will be enabled automatically once your first backup is complete. Wait a few minutes or run a full sync in the meantime..'
```

Comment:
Just noticed two dots at the end here. I suspect that it is a typo:

```suggestion
										'Selecting individual items to sync will be enabled automatically once your first backup is complete. Wait a few minutes or run a full sync in the meantime.'
```

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/105158/commits/77317a71d84f6557034b10d768ca44654516e37f